index.html

```
@@ -0,0 +1,850 @@
+        </div>
+        
+        <!-- Mobile menu -->
+        <div id="mobile-menu" class="md:hidden hidden border-t border-gray-200">
+            <div class="px-2 pt-2 pb-3 space-y-1">
+                <button onclick="showPage('home')" class="nav-link-mobile block px-3 py-2 text-base font-medium text-gray-900 hover:text-farm-green w-full text-left">Home</button>
+                <button onclick="showPage('team')" class="nav-link-mobile block px-3 py-2 text-base font-medium text-gray-500 hover:text-farm-green w-full text-left">Team</button>
+                <button onclick="showPage('contact')" class="nav-link-mobile block px-3 py-2 text-base font-medium text-gray-500 hover:text-farm-green w-full text-left">Contact</button>
+                <button onclick="showPage('faq')" class="nav-link-mobile block px-3 py-2 text-base font-medium text-gray-500 hover:text-farm-green w-full text-left">FAQ</button>
+            </div>
+        </div>
+            </div>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)